### PR TITLE
docs: added _Setting Podman machine default connection_

### LINKED ISSUE #3045: Update the troubleshooting guide
- [ ] Update outdated info https://github.com/containers/podman-desktop/pull/3088 https://github.com/containers/podman-desktop/pull/3089
- [ ] Should take the things we suggest to issues into account when talking with users and helping debug in the troubleshooting guide
- [x] Add troubleshooting info for: https://github.com/containers/podman/issues/19554
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/2806
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/1633
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/2794
- [x] Should update the troubleshooting guide with the new troubleshooting page feature https://github.com/containers/podman-desktop/pull/3083

### PR DESCRIPTION
**Pull request by themr0c**
Setting Podman machine default connection

### What issues does this PR fix or reference?
- references #2806
- references #3045

### CODE REVIEW

File: website/docs/getting-started/switching-podman-machine-default-connection.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 10
+title: Setting Podman machine default connection
+description: How to set Podman machine default connection.
+---
+
+## Setting Podman machine default connection
+
+Each Podman machine is exposing two connections:
```
Comment (cdrage) [inline website/docs/getting-started/switching-podman-machine-default-connection.md]:
```suggestion
Each Podman machine exposes two connections:
```

Sounds a bit more natural

File: website/docs/getting-started/switching-podman-machine-default-connection.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- rootless
+- rootful
+
+Podman has one default connection.
+
+Podman Desktop, and other tools, such as Kind, connect to the default connection.
```
Comment (cdrage) [inline website/docs/getting-started/switching-podman-machine-default-connection.md:16]: should it be Kind or `kind` ?

**Review by cdrage — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/switching-podman-machine-default-connection.md (hunk 2 above)
Comment (themr0c) [reply website/docs/getting-started/switching-podman-machine-default-connection.md:16]: We already settled on Kind, see: https://podman-desktop.io/docs/kubernetes/kind

**Comment by afbjorklund** [comment]
Does Podman Desktop even use the `podman system connection` ? I thought it only used unix sockets.

**Review by cdrage — APPROVED** [review]

**Pull request merged**